frontend/index.html

```diff
@@ -0,0 +1,17 @@
+<!doctype html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
Icon should be changed. Also, remove any mention of Vue or vite whether it's an image or text.